### ISSUE TITLE
crash when comment two lines at beginning of file

### ISSUE THREAD

**Issue by re-to** [spyder-ide/spyder#7959]
### What steps will reproduce the problem?

1. make a new file or use any .py file which starts with:
import os
import sys

2. then mark both first lines (parts!) and press Ctrl+1 to comment/uncomment
![image](https://user-images.githubusercontent.com/43666315/46193504-b0e3c900-c2fe-11e8-8a2c-755bb8490a4b.png)

3. this will result in hanging/crashing of spyder 3.3.1

## Versions

* Spyder version: 3.3.1 
* Python version: 3.6.6
* Qt version: 5.9.3
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.8.0 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.12.1 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.15.1 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.4.1 (OK)
IPython >=4.0     :  6.5.0 (OK)
matplotlib >=2.0.0:  2.2.3 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

**Comment by jnsebgosselin** [comment]
Thank you for your bug report.

I can reproduce this in Master also. There is no error, no traceback. Spyder simply stops responding.

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll take a look at this in a future release.

**Comment by jitseniesen** [comment]
This is now fixed as discussed in #11148.
